### ISSUE TITLE
Build failed on MSYS2 due to bad gcc

### ISSUE THREAD

**Issue by osm0sis** [mingwandroid/ctng-firefox-builds#2]
With a current, fully updated MSYS2 install it'll fail first with the downloaded gcc not matching the stored sha1um, then upon swapping in the updated package's sha1sum to get past the check it fails later during a configure step with "testing if gcc works.. no"

I've been struggling for a few days trying to get crosstools-ng from diorcety's darwin-target or master branch working (LLVM currently fails with "operating system not supported" for any version that doesn't rely on CMake there, and the CMake ones of course fail outright).

Any help would be much appreciated.

**Comment by mingwandroid** [comment]
Well, it used to work from the master branch a few years ago when I last tried it. I will not have time to look into this now, are you trying to build macOS targeting cross compilers here?

**Comment by mingwandroid** [comment]
The latest incarnation of this work is now part of the `Anaconda Distribution`. You can find the recipe [here](https://github.com/AnacondaRecipes/aggregate/tree/master/llvm-compilers-feedstock/recipe).

It will not work on Windows (or even Linux) at present. Although it isn't a *huge* amount of work for me to fix that (again) I will not have time to look into this.

PRs welcome, but fair warning: you will need to get familiar with `conda` and PRs that make the scripts or patches significantly more complicated or break macOS-native builds will be rejected.

**Comment by osm0sis** [comment]
Yes I am trying to target macOS for cross-compiling from MSYS2. I don't know where to begin with that "gcc works.. no" config issue.. sounds like it's maybe using the wrong gcc or some kind of issue with mixed paths..

I've hacked my way through a crosstool-ng build for Cygwin to Linux cross-compilers before, but this MSYS2 "operating system not supported" issue really has me stumped; I even tried hacking msys into LLVM 3.4's configure as an analogue for Cygwin on one attempt and MinGW on another with no change at all in output.

I won't even venture it with conda, I'll just hope you can find some time to give it a once-over at some point. Thanks for your time. :)
